ATB.BL/DataAccess/FileFlightRepository.cs

```diff
@@ -0,0 +1,77 @@
+﻿using ATB.Entities;
+
+namespace ATB.DataAccess
+{
+    internal class FileFlightRepository : IFlightRepository
+    {
+        private string flightsFilePath = "files/Flights.csv";
+
+        private Dictionary<(int, FlightClass), Flight> flightDictionary = new();   // (id,class) -> Flight       
+
+        public FileFlightRepository()
+        {
+            InitializeFlightDictionary();
+        }
+
+        public void AddAllFlights(IEnumerable<Flight> flights)
+        {
+
+            CsvUtility.AppendFlightsToCsv(flightsFilePath, flights);
+
+        }
+        public IEnumerable<Flight> GetAllFlights()
+        {
+            return flightDictionary.Values;
+        }
+
+        public Flight? GetFlight(int flightId, FlightClass flightClass) // returns null if the Flight is not in the dictionary
+        {
+            if(flightDictionary.TryGetValue((flightId, flightClass), out Flight flight))
+            {
+                return flight;
+            }
+            return null; 
```

Comment:
could you do this without an if? if yes, how?

Reply:
return flightDictionary.TryGetValue((flightId, flightClass), out var flight) ? flight : null;